src/DateTime/tests/test_datetime.py

```diff
@@ -220,13 +220,13 @@ def test_compare_methods(self):
 
     def test_compare_methods_none(self):
         # Compare a date to None
-        dt = DateTime('1997/1/1')
-        self.assertTrue(dt.greaterThan(None))
-        self.assertTrue(dt.greaterThanEqualTo(None))
-        self.assertFalse(dt.lessThan(None))
-        self.assertFalse(dt.lessThanEqualTo(None))
-        self.assertTrue(dt.notEqualTo(None))
-        self.assertFalse(dt.equalTo(None))
+        for dt in (DateTime('1997/1/1'), DateTime(0)):
+          self.assertTrue(dt.greaterThan(None))
+          self.assertTrue(dt.greaterThanEqualTo(None))
+          self.assertFalse(dt.lessThan(None))
+          self.assertFalse(dt.lessThanEqualTo(None))
+          self.assertTrue(dt.notEqualTo(None))
+          self.assertFalse(dt.equalTo(None))
```

Comment:
Please use the standard 4 space indentation.

Reply:
oops, you're right, thanks ! fixed.